WebApp/handler/SiteSummaryHandler.py

```diff
@@ -77,73 +47,6 @@ def web_getSiteSummaryData(self):
 
     self.finish(result)
 
-  @asyncGen
-  def web_action(self):
-
-    requestParams = self.__requestParams()
-    if 'action' in requestParams and requestParams['action']:
-
-      # pylint does not understand the action entry is not None any more
-      actionName = requestParams['action'][0]  # pylint: disable=unsubscriptable-object
-
-      methodName = actionName
-      if not actionName.startswith('set'):
-        methodName = '_get%s' % actionName
-
-      try:
-        result = yield self.threadTask(getattr(self, methodName), requestParams)
-      except AttributeError:
-        result = {'success': 'false', 'error': 'bad action %s' % actionName}
-
-    else:
-
-      result = {'success': 'false', 'error': 'Missing action'}
-
-    self.finish(result)
-
-  def _getHistory(self, requestParams):
```

Comment:
Is this method no longer used?

Reply:
These 2 methods are used, simply they inherit from ResourceSummaryHandler (the 2 pages are very similar indeed!)